**Background**

In quality control processes, especially when dealing with high-value items, destructive sampling is a necessary but costly method to ensure product quality. The test to determine whether an item meets the quality standards destroys the item, leading to the requirement of small sample sizes due to cost constraints.

**Scenario**

A manufacturer of print-heads for personal computers is interested in estimating the mean durability of their print-heads in terms of the number of characters printed before failure. To assess this, the manufacturer conducts a study on a small sample of print-heads due to the destructive nature of the testing process.

**Data**

A total of 15 print-heads were randomly selected and tested until failure. The durability of each print-head (in millions of characters) was recorded as follows:

1.13, 1.55, 1.43, 0.92, 1.25, 1.36, 1.32, 0.85, 1.07, 1.48, 1.20, 1.33, 1.18, 1.22, 1.29


In [44]:
import numpy as np
import scipy.stats as stats

In [45]:
sample=[1.13, 1.55, 1.43, 0.92, 1.25, 1.36, 1.32, 0.85, 1.07, 1.48, 1.20, 1.33, 1.18, 1.22, 1.29]
sample_mean=np.mean(sample)
sample_mean

1.2386666666666666

In [46]:
n=15 # no of samples

**Assignment Tasks**

**a. Build 99% Confidence Interval Using Sample Standard Deviation**
Assuming the sample is representative of the population, construct a 99% confidence interval for the mean number of characters printed before the print-head fails using the sample standard deviation. Explain the steps you take and the rationale behind using the t-distribution for this task.


In [47]:
n=15
s=np.std(sample,ddof=1) # standard deviation of sample
# confidence interval=0.99
# alpha=0.005

# t_test with confidence interval= (x_bar +- tcritical *(sample_sd/np.sqrt(n))


In [48]:
#tcritical
df=n-1
t_critical=stats.t.ppf(0.995,df)
t_critical

2.976842734370834

In [49]:
# construct the confidence interval
lower_limit_t=sample_mean-t_critical*(s/np.sqrt(n))
lower_limit_t

1.0901973384384906

In [50]:
upper_limit_t=sample_mean+t_critical*(s/np.sqrt(n))
upper_limit_t

1.3871359948948425

So,the 99% confidence interval for the mean number of characters printed before the print-head fails using the sample standard deviation
lies between **1.09 million and 1.38 million.**

**b. Build 99% Confidence Interval Using Known Population Standard Deviation**

If it were known that the population standard deviation is 0.2 million characters, construct a 99% confidence interval for the mean number of characters printed before failure.


In [51]:

n=15 # no of samples
pop_sd=0.2
# Confidence interval=0.99
# alpha=0.005

# Z-test with confidence interval=(x_bar +- zcritical *(pop_sd/np.sqrt(n)))



In [52]:
# z_critical
z_critical=stats.norm.ppf(0.995)
z_critical

2.5758293035489004

In [53]:
#construct the confidence interval
lower_limit_z=(sample_mean-z_critical*(pop_sd/np.sqrt(n)))
lower_limit_z

1.1056514133957607

In [54]:
upper_limit_z=(sample_mean+z_critical*(pop_sd/np.sqrt(n)))
upper_limit_z

1.3716819199375725

So, the 99% confidence interval for the mean number of characters printed before failure lies betwween **1.11 million and 1.37 million.**